### ISSUE TITLE
Skip prefixed properties

### ISSUE THREAD

**Issue by curvedmark** [curvedmark/roole#16]
```
body
    -webkit-border-radius: 10px
    border-radius: 20px
```

compiles to

```
body {
    -webkit-border-radius: 10px
    -moz-border-radius: 20px
    border-radius: 20px
}
```

But may not be a good idea to skip prefixed property values like `linear-gradient()`.

**Comment by curvedmark** [comment]
not sure about this one, is the performance penalty of looking for previous property really worth it? I guess most user won't use this feature, yet the penalty applies to everyone. Providing an option?

Closing for now until real-world use cases come up.

**Comment by curvedmark** [comment]
Landed in ab37bdcfc7e834ee0023467f59068af8e43d4690
